shared/teams/team/rows/invite-row/request/index.tsx

```diff
@@ -59,21 +55,88 @@ export const TeamRequestRow = (props: Props) => {
           open={props.isRolePickerOpen}
           disabledRoles={props.disabledReasonsForRolePicker}
         >
-          <Button label="Let in as..." onClick={onAccept} small={true} style={styles.letInButton} />
+          <Kb.Button label="Let in as..." onClick={onAccept} small={true} style={styles.letInButton} />
         </FloatingRolePicker>
-        <Button
+        <Kb.Button
           label="Ignore"
           onClick={onIgnoreRequest}
           small={true}
           style={styles.ignoreButton}
           type="Danger"
         />
-        {!Styles.isMobile && <Icon onClick={onChat} style={styles.icon} type="iconfont-chat" />}
-      </Box>
-    </Box>
+        {!Styles.isMobile && <Kb.Icon onClick={onChat} style={styles.icon} type="iconfont-chat" />}
+      </Kb.Box>
+    </Kb.Box>
   )
 }
 
+const TeamRequestRowNew = (props: Props) => {
+  const {ctime, fullName, username, onAccept, onOpenProfile} = props
+
+  const {showingPopup, setShowingPopup, toggleShowingPopup, popup, popupAnchor} = Kb.usePopup(attachTo => (
+    <Kb.FloatingMenu
+      items={[
+        {onClick: props.onChat, title: 'Chat'},
```

Comment:
We put `icon` props on these now. https://zpl.io/V11NkLB

Reply:
Ah so that's where it ends up!